src/Avalonia.Controls/Platform/IWindowBaseImpl.cs

```diff
@@ -65,7 +65,27 @@ public interface IWindowBaseImpl : ITopLevelImpl
         /// Sets the client size of the toplevel.
         /// </summary>
         void Resize(Size clientSize);
-        
+
+        /// <summary>
+        /// Minimum width of the window.
+        /// </summary>
+        double MinWidth { get; set; }
```

Comment:
Might it be better to use the `Size` struct here so that each implementor needs to only add two properties instead of four?  `MinSize` and `MaxSize`.

Reply:
i copied it from some layout classes, i have no preference so its up to you ... will try to fix GTK build first than will do cosmetics

Reply:
I think `Size` would make sense here as these variables aren't used in binding or XAML anything.

Reply:
i've replaced these properties with one function SetMinMaxSize, these values are transferred only form Window to particular WindowingPlatformImpl